dev/integration_tests/web/lib/web_directory_loading.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+Future<void> main() async {
```

Comment:
A brief explanation about how this test attempts to reproduce the use-case would be useful to future readers of this code.

Reply:
Done